boards/arm/mec1501modular_assy6885/pinmux.c

```diff
@@ -403,11 +403,11 @@ static int board_pinmux_init(struct device *dev)
 	pinmux_pin_set(portc, MCHP_GPIO_125, MCHP_GPIO_CTRL_MUX_F1);
 	pinmux_pin_set(portc, MCHP_GPIO_121, MCHP_GPIO_CTRL_MUX_F1);
 	pinmux_pin_set(portc, MCHP_GPIO_122, MCHP_GPIO_CTRL_MUX_F1);
-#if DT_SPI_XEC_QMSPI_0_LINES == 4
+#if DT_INST_PROP(0, lines) == 4
 	pinmux_pin_set(portc, MCHP_GPIO_123, MCHP_GPIO_CTRL_MUX_F1);
 	pinmux_pin_set(portc, MCHP_GPIO_126, MCHP_GPIO_CTRL_MUX_F1);
 #endif
-#endif /* DT_SPI_XEC_QMSPI_0_PORT_SEL == 0 */
+#endif /* DT_INST_PROP(0, port_sel) == 0 */
```

Comment:
These should be something like `DT_PROP(DT_INST(0,microchip_xec_qmspi), port_sel)`